experimental/packages/opentelemetry-instrumentation-grpc/src/instrumentation.ts

```diff
@@ -50,19 +51,29 @@ import {
   SpanKind,
   trace,
   Span,
+  type Attributes,
 } from '@opentelemetry/api';
 import {
   InstrumentationNodeModuleDefinition,
   InstrumentationBase,
 } from '@opentelemetry/instrumentation';
+
+// stable http attributes
+// net.peer.name → server.address
+// net.peer.port → server.port
```

Comment:
I *assume* the world is all UTF-8 and can handle non-ASCII chars in source files. :)

Reply:
Heh I forgot that was still in there, removing.